### DISCUSSION TITLE
Deploy waring

### DISCUSSION THREAD

**Discussion by windmx** [arthelokyo/astrowind#357] [Q&A] [answered]
Warning: Detected "engines": { "node": ">=18.14.1" } in your `package.json` that will automatically upgrade when a new major Node.js Version is released. Learn More: http://vercel.link/node-version

**Comment by thkruz** [comment, marked as answer]
This really is more of an issue with Vercel than this repo.

It is letting you know that at some point in the future Vercel will default to Node 20 vs Node 18 and that your package.json allows it switch automatically. I am running Node 20 without any issues, so it looks like a false positive and the solution to hiding the warning is going to be specific to Vercel - try changing >= to == to let Vercel know not to upgrade to Node 20 automatically.

Recommend you close this issue.
